src/Common/ILoggerExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Oryx.Common
+{
+    /// <summary>
+    /// Extensions to ASP.NET Core's ILogger
+    /// </summary>
+    public static class ILoggerExtensions
+    {
+        /// <summary>
+        /// Logs message and the supplied properties. The message must not have formatted strings as they are not processed.
+        /// </summary>
+        /// <param name="logger"></param>
+        /// <param name="logLevel"></param>
+        /// <param name="nonFormattedMessage">The message without any formatting strings in it.</param>
+        /// <param name="properties"></param>
+        public static void Log(
```

Comment:
is this used anywhere?

Reply:
It is used by the `LogLongMessage` extension...the `properties` argument should make it clear..

Reply:
oh, totally missed it